main.go

```diff
@@ -397,10 +390,11 @@ func configureSAML() error {
 	return nil
 }
 
-func BestDomain() string {
-	domain := config.FindInfo().Domain
-	if domain != "" {
-		return domain
-	}
-	return httpHost
-}
+// The following is unused not removing just commenting out
```

Comment:
curious as to what the original intent of this was

Reply:
Okay I found out what the original intent was by looking at https://github.com/frequencyanalytics/frequency/blob/master/main.go another project by cloud.app 

TLDR this should have been used to find the domain however it as replaced by the httpHost lookup as part of doing cookie auth. 

So this whole thing can be killed.